META

```diff
@@ -0,0 +1,36 @@
+description = "Coq Proof Assistant Plugin API"
+version = "dev"
+
+requires = "camlp5"
```

Comment:
Indeed this META file is barely a sketch, but for that particular purpose I wouldn't support camlp4 anymore.

By the way, I would avoid automatic generation of META files. `exists_if` and other conditional constructs exist there to handle variable cases (for instance you can do:
```
package "ide" (
    description = "Coq Ide Libraies"
    version = "[distributed with Coq]"
    exists_if = "ide/idelib.cmxa"
...
}